README.md

```diff
@@ -174,6 +174,24 @@ summary.get({ service: 'database' })
 # => { 0.5 => 0.1233122, 0.9 => 3.4323, 0.99 => 5.3428231 }
 ```
 
+## Releases
+
+When a new version of this gem is released, the tags within the gemfile in the following repositories must be updated.
+
+* [Akon client](https://github.com/CrowdFlower/akon_client)
```

Comment:
I didn't see this dependency in Akon Client. Maybe we mean Akon?

Reply:
Ahhh you are right! Thank you for noticing! I'll change it in the others too.